en-US/S.xml

```diff
@@ -326,17 +326,31 @@
 
 			</listitem>
 
-<!-- Added entries for "sign in" and "sign in to". -->
 		</varlistentry>
 		 <varlistentry id="sign-in">
-			<term>sign in</term>
+			<term>sign-in, sign in</term>
```

Comment:
Similar to the entry for "log in", I suggest adding an entry with the preferred preposition  - I'm assuming "sign in to".